queries/wit/highlights.scm

```diff
@@ -0,0 +1,84 @@
+; Comments
+(line_comment) @comment
+
+(block_comment) @comment
+
+; Primitive Types
+[
+  "bool"
+  "s8"
+  "s16"
+  "s32"
+  "s64"
+  "u8"
+  "u16"
+  "u32"
+  "u64"
+  "float32"
+  "float64"
+  "char"
+  "string"
+] @type.builtin
+
+; Container Types
+[
```

Comment:
We can group these two into one and have comments separating them

Reply:
Ok, amended ✅